### PR TITLE
update news feed8/10

### PR DESCRIPTION
**Pull request by johngruen**

### CODE REVIEW

File: content-src/news_updates.yaml
Comment (chuckharmston) [inline content-src/news_updates.yaml:12]: Did you mean to delete these?

Comment (johngruen) [reply content-src/news_updates.yaml:12]: yeah, they don't expire on their own ATM

**Review by chuckharmston — APPROVED** [review]
I'll let you self-merge after you've confirmed that you wanted to delete the other news updates.

(And after CI passes.)

File: content-src/news_updates.yaml
Comment (chuckharmston) [reply content-src/news_updates.yaml:12]: I'm not sure we should do this; there will be l10n conflicts in the future when we add new ones.

File: locales/en-US/experiments.ftl
Comment (lmorchard) [inline locales/en-US/experiments.ftl:255]: FWIW, [we need to wait to get some l10n review whenever changing or adding strings](https://github.com/mozilla/testpilot/issues/2574), so this shouldn't be just a quick merge. That's why we have the auto-request for review from @flodolo up there

File: content-src/news_updates.yaml
Comment (lmorchard) [reply content-src/news_updates.yaml:12]: It's always been the plan to remove news updates from the YAML. [The slug (which should be unique) is a part of the l10n IDs](https://github.com/mozilla/testpilot/blob/master/frontend/tasks/util.js#L63) - that should help prevent conflicts

File: content-src/experiments/send.yaml
Comment (lmorchard) [inline content-src/experiments/send.yaml:87]: In fact, to keep these things extra unique, maybe stick a date in there - like `send-update-2017-08-10-1`. These slugs need to never be accidentally re-used with new updates later on when old ones have been long since deleted.

**Review by flodolo — APPROVED** [review]
I'm OK with these changes, but I think we need to start a larger discussion on how to manage this section.

They're not going to be localized, so I wonder if we could find a less prominent space to show them for language != en-US (is it just me or they're not displayed right now on TP?), and only localize a section title (TP Latest News) + blurb explaining that news are in English, and only show the title for each news in English.

**Comment by lmorchard** [comment]
> They're not going to be localized

@flodolo <strike>Why aren't these strings going to be localized?</strike> Oh! I see what you mean: The *strings* in the news items on the site are localized, but the Medium blog posts aren't. Good point.

**Comment by lmorchard** [comment]
Submitted issue #2762, but otherwise this looks like it's got all the r+'s

**Pull request merged**
